packages/experimental/Shadow/src/Shadow.tsx

```diff
@@ -92,51 +92,53 @@ function getStylePropsForShadowViewsWorker(childStyleProps: ViewStyle = {}, shad
   return {
     inner: {
       style: {
-        borderBottomWidth,
-        borderEndWidth,
-        borderLeftWidth,
-        borderRightWidth,
-        borderStartWidth,
-        borderTopWidth,
-        borderWidth,
-
-        padding,
-        paddingBottom,
-        paddingEnd,
-        paddingHorizontal,
-        paddingLeft,
-        paddingRight,
-        paddingStart,
-        paddingTop,
-        paddingVertical,
-
-        alignItems,
-
-        flexWrap,
-        flexDirection,
+        // Only add the prop if it is defined, otherwise every prop will be added and be set to
+        // undefined, which can cause unexpected behaviour with some of the styles
+        ...(borderBottomWidth && { borderBottomWidth }),
```

Comment:
I wonder there's a better way to do this, but I am not sure 😅. I'm not sure of the perf implications of so many spread operators. Maybe there's something like `mergeStyle` that's useful in `@fluentui-react-native/framework`?

Not opposed though, I think this is a good overall change :)

Reply:
Ended up taking out all the undefined props after creating the shadow inner/outer style objects - if you had a different idea in mind let me know.